### ISSUE TITLE
No way to check if the script worked. 

### ISSUE THREAD

**Issue by sifotes** [lacasse/Smart-Heater#2]
I'm assuming the smartplug returns a response of some sort after processing the commands sent. 

It would be helpful to add a listener to the script in order to echo back a confirmation of the command.

**Comment by lacasse** [comment]
You're right. The only way to tell if it's working at the moment is to be there physically. Since I have a rf receiver as well at the transmitter, I'll add a functionality that validates if signals are really being sent.
